inc/admin.php

```diff
@@ -299,7 +299,7 @@ function ppom_admin_save_form_meta() {
 			'message'        => __( 'Form added successfully', 'woocommerce-product-addon' ),
 			'status'         => 'success',
 			'productmeta_id' => $ppom_id,
-			'redirect_to'    => $redirect_to,
+			'redirect_to'    => esc_url( $redirect_to ),
 		);
```

Comment:
can you please use `esc_url_raw` here and for other `redirect_to` params in AJAX responses. JS uses that URL to redirect the page.

That's valid for all `redirect_to` params in the AJAX responses, as I see some of them are missing by esc.

![Screen Shot 2023-05-03 at 10 02 02](https://user-images.githubusercontent.com/25361626/235850833-8d3961ee-7205-4d50-b602-6a76e99d962b.jpg)

Reply:
P.S.: as I see `js/admin/ppom-admin.js` handles the redirection with the AJAX response.

Reply:
@mskapusuz Fixed, Thank you!